BeVolt/App/src/main.c

```diff
@@ -1344,4 +1347,32 @@ int main(void){
 	while(1){}
 }
 
+#elif defined DashboardMessage
+
+void DashboardMessageTest(){
```

Comment:
This looks pretty reasonable. This doesn't do much for us once we've merged it into the master branch though. It would probably be good to actually incorporate this into our real main function in some way (maybe as a function that can be called from `main()`. What you have here would probably be good enough if you moved it out of the `#ifdef` and into say `StatusSync.c` to be called from main.